includes/utilities.inc

```diff
@@ -505,13 +505,13 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
 
     $updated_display_values
       = "<span class='toggle-wrapper'>" .
-      t("<span>!value !separator !show_more</span>", array(
+      t("<span>!value !show_more</span>", array(
         '!separator' => $separator,
```

Comment:
@jordandukart is right but also you aren't changing how the `$truncated_value` is being created so why touch this at all?